frontend/src/console/components/Logos/displayMessages.ts

```diff
@@ -27,6 +27,14 @@ const messages = {
       "If unset, OpenCraft's logo will be used.",
     description: 'Favicon explanation'
   },
+  footerLogoImageSrc: {
+    defaultMessage: 'Footer Logo Image URL',
+    description: "The URL of the image for the footer's logo"
```

Comment:
nit: single and double quotes mixing.

Reply:
I did it to avoid having to escape quotes; it also seems to be the style used by other messages in this file but let me know if you still want it changed.